packages/round-manager/src/context/application/usePayout.ts

```diff
@@ -124,34 +130,77 @@ export function usePayout() {
       setContractApproveSpendStatus(ProgressStatus.IS_SUCCESS);
     }
 
+    console.log("Payout 1");
+
     // Payout
-    const directPayout = DirectPayoutStrategy__factory.connect(
-      payoutStrategyAddress,
-      signer
-    );
-    let txBlockNumber: number;
+    // const directPayout = DirectPayoutStrategy__factory.connect(
+    //   roundId,
+    //   signer
+    // );
+    // let txBlockNumber: number;
     try {
-      setContractUpdatingStatus(ProgressStatus.IN_PROGRESS);
-      const payoutReceipt = await directPayout.payout({
-        vault: payoutVault,
+      console.log("Payout 2");
+      await allo.payoutDirectGrants({
```

Comment:
and the editor points out that await is not required here (which is weird cause we do an await on all other functions in allo.ts)

Reply:
you need to execute the operation by running `execute`, check other calls to Allo.